app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
```

Comment:
- [ ] It's good practice if you put `update_likes_counter` after the private method. And call it `after_create`.
[More info...](https://guides.rubyonrails.org/active_record_callbacks.html#:~:text=before_validation%20%3Aensure_login_has_a_value-,private,-def%20ensure_login_has_a_value%0A%20%20%20%20%20%20if)